doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -23,6 +23,111 @@ Distribution
 
 The |ISE| is provided as a precompiled binary, which is part of the nRF54H20 SoC bundle and is provided independently from the |NCS| release cycle.
 
+Installing
+**********
+
+For new setups, follow the instructions found in :ref:`ug_nrf54h20_gs`.
+
+Updating IronSide SE
+********************
+
+Brief instruction
+=================
+
+To update the IronSide SE firmware, use the ``west ncs-ironside-se-update`` command with the following syntax:
+
+.. code-block:: console
+
+   west ncs-ironside-se-update --zip <path_to_soc_binaries.zip> --allow-erase
```

Comment:
add another command right after, that updates the rslot.

So the user gets the correcrt impression that he should update both.